### ISSUE TITLE
list of org-static-blog sites?

### ISSUE THREAD

**Issue by summaminutiae** [bastibe/org-static-blog#19]
I'm enjoying the ease of blogging with org-static-blog, but of course now I want to tinker with the css styles.  Is there a list of blogs that use this, or some unique string in o-s-b pages I could search for?

**Comment by bastibe** [comment]
No, sorry, no such list exists, and org-static-blog does not include any kind of searchable metadata.

I know of https://bastibe.de/ and https://zngguvnf.org/. If you like, you can create a Pull Request for a new section in the README and add your own blog, there! That would be really cool!

**Comment by neeasade** [comment]
I found some with this search[1]. Maybe there could be a wiki page or something. 

[1] https://github.com/search?q=setq+org-static-blog-publish-url&type=Code

**Comment by matthewbauer** [comment]
My blog is published with org-static-blog here:

https://matthewbauer.us/blog/

Styling is pretty minimalist right now.

**Comment by bastibe** [comment]
Hey, thank you for sharing!

I added your blog to the README, and added a note for other users, to create a pull request if they want to be added to the README. (I wouldn't want to add people's blogs here without their express permission.)

**Comment by alhassy** [comment]
Hey @bastibe, the MWE really helped! With it, I got my blog going: http://alhassy.github.io/

Notable things include:
+ ✓ A nice blog banner
+ ✓ Overall nice looking HTML style
+ ✓ A floating, yet unobtrusive, table of contents
+ ✓ Comments for blog readers --- Based on one of your blog posts, thanks!
+ ✓ Articles have dedicated images which are displayed
  on the blog's welcome page along with the article's abstract
+ ✓ Tooltips, folded regions, and badges
+ ✓ Curvy style for code and tables
+ ✓ Article footers link to the [colourised] Org source and to the Github history

The write-up on how I set these things up can be found at https://alhassy.github.io/AlBasmala

**Comment by bastibe** [comment]
Thank you, cool blog you built there!

Would you like to add it to the blog list in the README with a pull request?

By the way, I was initially very confused by your blog, until I noticed that Firefox chose a spectacularly poor replacement font for "fantasy":

![Screenshot_20200505_082931](https://user-images.githubusercontent.com/649181/81040044-c6da4800-8eaa-11ea-83be-636ef07a87e4.png)

(Firefox on top, Chrome on bottom)

**Comment by alhassy** [comment]
bahahaha that's insane! ....maybe I should reconsider `fantasy`, but I really do like it ^_^

Please do add it; thanks.

Perhaps, you could help me get rid of that `<br>` in the screenshot above?

**Comment by bastibe** [comment]
You can create the pull request for adding your blog right here on Github, by clicking the little pen symbol on the README.org file.

The `<br>` is in your HTML title.
